docs/cli.md

```diff
@@ -180,24 +180,48 @@ This command will create a bare-bones sample file for one of the top level OSCAL
 ...
 ```
 
-The following subcommands are currently supported:
+The `-t` specifies the type of the model to create, which can be one of catalog, profile, component-definition, system-security-plan, assessment-result, plan-of-action-and-milestone.  Each type will be created in its corresponding directory, such as catalogs, profiles, etc.
 
```

Comment:
after ssp, "assessment-plan" should be there if we are specifying all types of model.